### DISCUSSION TITLE
Getting Error for this  "aries-bifold": "workspace:^" dependency

### DISCUSSION THREAD

**Discussion by Ankit2598** [bcgov/bc-wallet-mobile#1570] [Q&A]
Hi Team

I am getting error " Error: aries-bifold@workspace:^: Workspace not found (aries-bifold@workspace:^)" when I am running yarn install command on the root directory to install all the dependency.

"aries-bifold": "workspace:^" dependency is present under app/package.json file.

Can you please guide me to fix this.

**Comment by cvarjao** [comment]
make sure the [bifold-core](https://github.com/bcgov/bc-wallet-mobile/blob/main/bifold-core) symlink is being created properly. Are you on a windows or linux/mac computer?

**Reply by Ankit2598** [reply]
Thanks @cvarjao for the help
I am using windows 10. 
Bifold-core symlink was created and the content was "./bifold/packages/legacy/core" same as you have have shared above.

Actually I am new to this, can you please brief about the steps about how can I create bifold-core symlink properly so I will recheck the steps to find the issue.

my yarn version:3.3.1
my node version:16.15.0

**Reply by cvarjao** [reply]
it should have no content, it should be a symbolic link to a directory. Make sure your git has symlink support enabled:
https://stackoverflow.com/questions/5917249/git-symbolic-links-in-windows/59761201#59761201

**Comment by Ankit2598** [comment]
@cvarjao 

➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: aries-bifold@workspace:^: Workspace not found (aries-bifold@workspace:^)
    at He.getWorkspaceByDescriptor (C:\Users\e02619\Desktop\BC Wallet\bc-wallet-mobile-old\.yarn\releases\yarn-3.3.1.cjs:439:3232)
    at FC.getCandidates (C:\Users\e02619\Desktop\BC Wallet\bc-wallet-mobile-old\.yarn\releases\yarn-3.3.1.cjs:390:27510)
    at zf.getCandidates (C:\Users\e02619\Desktop\BC Wallet\bc-wallet-mobile-old\.yarn\releases\yarn-3.3.1.cjs:391:1264)
    at zf.getCandidates (C:\Users\e02619\Desktop\BC Wallet\bc-wallet-mobile-old\.yarn\releases\yarn-3.3.1.cjs:391:1264)
    at C:\Users\e02619\Desktop\BC Wallet\bc-wallet-mobile-old\.yarn\releases\yarn-3.3.1.cjs:439:7695
    at Ff (C:\Users\e02619\Desktop\BC Wallet\bc-wallet-mobile-old\.yarn\releases\yarn-3.3.1.cjs:390:8965)
    at ge (C:\Users\e02619\Desktop\BC Wallet\bc-wallet-mobile-old\.yarn\releases\yarn-3.3.1.cjs:439:7675)
➤ YN0000: └ Completed in 3s 552ms
➤ YN0000: Failed with errors in 3s 560ms

**I am getting the above error while running yarn install command.**


npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:^

**I am getting the above error while running npm install.**

 **I also checked that my git has symlink supported.**

Can you please suggest me how to solve this.

**Reply by cvarjao** [reply]
[bifold-core](https://github.com/bcgov/bc-wallet-mobile/blob/main/bifold-core) needs to be a symbolic to `./bifold/packages/legacy/core`. Make sure you `git` is setup to handle symbolic links. Check if the `bifold-core` folder exists it is properly linked. Or use a linux-based dev container.

**Comment by cvarjao** [comment]
We no longer use submodules or symbolic links
